### PR TITLE
[Feature store] Add timedelta & category support + fix timestamp column handling

### PR DESCRIPTION
**Pull request by urihoenig**

### CODE REVIEW

**Comment by urihoenig** [comment]
To be merged only after https://github.com/mlrun/storey/pull/185

File: tests/feature-store/test_feature_store.py [hunk 1]
```diff
@@ -44,10 +44,76 @@ def test_basic_featureset():
...
+    verify_ingest(data, "int")
+    verify_ingest(data, "int", infer=True)
+    verify_ingest(data, "float")
+    verify_ingest(data, "float", infer=True)
+    verify_ingest(data, "bool")
+    verify_ingest(data, "bool", infer=True)
+    verify_ingest(data, "timestamp")
+    verify_ingest(data, "timestamp", infer=True)
+    verify_ingest(data, "category")
+    verify_ingest(data, "category", infer=True)
```
Comment (Hedingber) [inline tests/feature-store/test_feature_store.py:106]: I see you're already supporting keys to be a list in `verify_ingest`, why not using it and give one key every time ?
Even if there is some reason for that, this code can be written in much more elegant way, e.g.:
```
for key in data.keys():
    verify_ingest(data, key, infer=False)
    verify_ingest(data, key, infer=True)
```

File: tests/feature-store/test_feature_store.py [hunk 2]
```diff
@@ -44,10 +44,76 @@ def test_basic_featureset():
...
+
+    # Timedelta isn't supported in parquet
+    data["timedelta"] = pd.Timedelta("-1 days 2 min 3us")
+
+    verify_ingest(data, "key", targets=[TargetTypes.nosql])
+    verify_ingest(data, "key", targets=[TargetTypes.nosql], infer=True)
+    verify_ingest(data, "timedelta", targets=[TargetTypes.nosql])
+    verify_ingest(data, "timedelta", targets=[TargetTypes.nosql], infer=True)
```
Comment (Hedingber) [inline tests/feature-store/test_feature_store.py:114]: Same here, use some loop or something

**Review by Hedingber — CHANGES_REQUESTED** [review]

File: tests/feature-store/test_feature_store.py (hunk 1 above)
Comment (urihoenig) [reply tests/feature-store/test_feature_store.py]: It's supported as feature store should support multi column keys (in progress).

**Review by Hedingber — APPROVED** [review]

**Pull request merged**